### ISSUE TITLE
Foobar x86 convert plugin fails

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rakibtg/gdipp#23]
```
What steps will reproduce the problem?
1. install latest 0.7 gdipp, service mode
2. foobar 1.0.1 with default convert plugin
3. convert any song to any format

What is the expected output? What do you see instead?
After pressing OK button, no action at all. After stopping the x86 service,
everything's back to normal.

What version of gdipp are you using? On what operating system (with
bitness)?
0.7 on windows 7 x64 ultimate with foobar 1.0.1

Please provide any additional information below.
I　guess that gidpp influences addon systems of foobar, chrome and firefox. 

```

Original issue reported on code.google.com by `DCatcher...@gmail.com` on 23 Mar 2010 at 2:24

**Comment by GoogleCodeExporter** [comment]
```
The problem is fixed with 0.7.3 build.
```

Original comment by `DCatcher...@gmail.com` on 7 Apr 2010 at 8:14

**Comment by GoogleCodeExporter** [comment]
Original comment by `crendk...@gmail.com` on 7 Apr 2010 at 10:20
- Changed state: **Fixed**
